cli/src/test/java/com/devonfw/tools/ide/context/ProcessContextTestImpl.java

```diff
@@ -21,7 +22,11 @@ public ProcessContextTestImpl(IdeContext context) {
 
   @Override
   public ProcessResult run(ProcessMode processMode) {
-
-    return super.run(ProcessMode.DEFAULT_CAPTURE);
+    ProcessResult result = super.run(ProcessMode.DEFAULT_CAPTURE);
+    // this hack is still required to capture test script output
+    if (processMode == ProcessMode.DEFAULT || processMode == ProcessMode.BACKGROUND) {
```

Comment:
this should only happen on success since in error case it was already logged.
```suggestion
if (result.isSuccessful() && (processMode == ProcessMode.DEFAULT || processMode == ProcessMode.BACKGROUND)) {
```